mxnet_model_service.py

```diff
@@ -0,0 +1,95 @@
+import mxnet as mx
+import numpy as np
+import tarfile
+import json
+import os
+
+from mxnet.gluon.utils import download
+from mxnet.io import DataBatch
+from model_service import SingleNodeModel, MultiNodesModel
+
+URL_PREFIX = ('http://', 'https://', 's3://')
```

Comment:
How about put URL_Prefix in model_service.py, I think it's shared by all model services.

Reply:
Agree.